docfx_project/ziti/clients/tunneler-overview.md

```diff
@@ -1,4 +1,7 @@
-A tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the Ziti Network.
-Ziti provides tunnelers for each major operating system.
+Ziti Tunnellers allow pre-existing, unmodified TCP/IP applications to access or provide services on a secure Ziti network. A Ziti Tunneler is a Ziti-native application that communicates with local peers using TCP/IP and proxies the payload to/from a Ziti service.

```

Comment:
The previous opening sentence was simpler and more approachable. I'd like to see it returned. "A tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the Ziti Network."

Only change we probably should make is to normalize to use OpenZiti instead of Ziti nowadays